libexec/rtld-elf/rtld_c18n.h

```diff
@@ -210,7 +211,7 @@ struct tramp_header {
 	const Obj_Entry *defobj;
 	size_t symnum;
 	struct func_sig sig;
-	uint32_t entry[];
+	_Alignas(INST_ALIGN) uint8_t entry[];
```

Comment:
Why? Just so you don't have to cast it to char for your arithmetic? uint32_t is arguably more proper on AArch64 given the instruction format...